src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+import { PaginationProps } from '../../types/PaginationProps';
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: PaginationProps) => {
+  const pageNumber = Math.ceil(total / perPage);
+  const generatePageItems = (pageAmount: number) => {
+    const pages = [];
+
+    for (let i = 1; i < pageAmount + 1; i++) {
+      pages.push(
+        <li
+          className={cn('page-item', {
+            active: currentPage === i,
+          })}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${i}`}
+            onClick={() => {
+              onPageChange(i);
+            }}
+          >
+            {i}
+          </a>
+        </li>,
+      );
+    }
+
+    return pages;
```

Comment:
you can just use the `getNumbers` function to generate the array

Reply:
Still not fixed